jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/TaskCommon.java

```diff
@@ -43,15 +43,18 @@
     if (!project.getPlugins().hasPlugin(WarPlugin.class)) {
       return null;
     }
+    return project.getTasks().named(WarPlugin.WAR_TASK_NAME);
+  }
 
+  @Nullable
+  static TaskProvider<Task> getBootWarTaskProvider(Project project) {
```

Comment:
Can we make the logic between `getWarTaskProvider()` and `getBootWarTaskProvider()` consistent? e.g. `getWarTaskProvider()` starts with `if (!project.getPlugins...)` and `getBootWarTaskProvider()` starts with `if(project.getPlugins...)`.

Reply:
I feel comfortable with this. This is at least the original behavior. I think the WAR Plugin is an integral part of Gradle (applied with `id 'war'`), so it is unthinkable that it doesn't have the `war` task or will remove the task in the future. I think that simply won't happen. OTOH, the Spring Boot plugin is a third-party and we can exercise caution.

And nice for catching CHANGELOG. This keeps occurring to me :/

Reply:
Oh, sorry. You were talking about something else.